cc: Update doc for Endpoints and Deploy API via APIM (#3804)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md

    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps [here](https://github.com/wso2/product-apim#installation--running).

### Step 1 - Find the API-M IP Address

> In order to tell Choreo Connect where API Manager (API-M) is located, find out the IP that can be used to access the API Manager instance. If you are trying out WSO2 API Manager locally, the private IP retrived using `hostname -I` or `ipconfig` would do.

Reviewer: ```suggestion
Choreo Connect identifies the location of the API-M server instance by its IP address. If you are trying out WSO2 API Manager locally, you can use the private IP retrieved using `hostname -I` or `ipconfig`.
```

Author: Updating this locally

### Step 2 - Update the Choreo Connect Config File

> Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update it as follows.

Reviewer: ```suggestion
Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file in a text editor and update the following configuration:
```

Author: Updating this locally

  skipSSLVerification=true
  # Message broker connection URL of the control plane
    [controlPlane.jmsConnectionParameters]
        eventListeningEndpoints = ["amqp://admin:$env{cp_admin_pwd}@192.168.8.101:5672?retries='10'&connectdelay='30'"]
 ``` 

!!! tip

    In API Manager, a new Gateway Environment can be created from the Admin Portal (available at `https:<apim-host>:<apim-port>/admin`) **Gateways** tab. For an example, if a new Gateway Environment was added with the name "choreo-connect" and display name "Choreo Connect", add the following changes to the `docker-compose.yaml` and `config.toml`.

    1. In `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/docker-compose.yaml` update the environment variables `ENFORCER_LABEL` and `ROUTER_LABEL` to `choreo-connect`
    2. In `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml`,
        i. update `environmentLabels` in `[controlPlane]` to `["choreo-connect"]`
        ii. to be able to use accept APIs via apictl, include "choreo-connect" to [[adapter.vhostMapping]] as shown below.
        ```
        [[adapter.vhostMapping]]
        environment = "choreo-connect"
        vhost = "localhost"
        ```

> ### Step 3 - Start the Choreo Connect

Reviewer: ```suggestion
### Step 3 - Start Choreo Connect
```
